kyaml/kio/byteio_reader.go

```diff
@@ -115,7 +114,8 @@ func (r *ByteReader) Read() ([]*yaml.RNode, error) {
 	if err != nil {
 		return nil, errors.Wrap(err)
 	}
-	values := strings.Split(input.String(), "\n---\n")
+
+	values := strings.Split(strings.Replace(input.String(), "\r\n", "\n", -1), "\n---\n")
```

Comment:
Maybe add a comment here to explain to future readers of this code why we are doing this.

Reply:
Done